spec/features/user_can_create_other_users_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.feature "Anyone using the service can create a user" do
+  context "As a person using the playbook" do
+    scenario "I can create a new user" do
+      visit pages_path
+
+      expect(page).to have_content("Sign Up")
+
+      click_link "Sign Up"
+
+      expect(page).to have_content("Create User")
+      expect(page).to have_content("Name")
+
+      fill_in "Name", with: "Name"
```

Comment:
```suggestion
      fill_in "Name", with: "Test user"
```

Better to test with something that would never be part of the defaults or the markup.